src/lang/modifyAst/boolean.ts

```diff
@@ -271,25 +256,26 @@ export function getLastVariable(
 
 export function booleanSubtractAstMod({
   ast,
-  targets,
+  solids,
   tools,
 }: {
   ast: Node<Program>
-  targets: VariableDeclaration[]
+  solids: VariableDeclaration[]
   tools: VariableDeclaration[]
 }): Node<Program> {
+  console.log('booleanSubtractAstMod', solids, tools)
   const newAst = structuredClone(ast)
   const newVarName = findUniqueName(newAst, 'solid')
   const createArrExpr = (varDecs: VariableDeclaration[]) =>
     createArrayExpression(
       varDecs.map((varDec) => createLocalName(varDec.declaration.id.name))
     )
-  const targetsArrayExpression = createArrExpr(targets)
+  const solidsArrayExpression = createArrExpr(solids)
   const toolsArrayExpression = createArrExpr(tools)
```

Comment:
Debatable here but just like sweeps, we could also add an if length == 1 check and not create an array in that case

Reply:
Oh is the kwarg able to take an array or a single value?

I thought it had to be an array.

Reply:
@Irev-Dev Yup, `solid001 = subtract(extrude001, tools = extrude002)` works perfectly fine! I'll make the change

Reply:
Done